### PR TITLE
Implemented New Mead-Portal Activation Flow

### LINKED ISSUE #2179: Implement Mead-Portal connection flow with activation code
- [x] Logic
  - [x] Mead Querying API
  - [x] Mead State Manage
  - [x] Error Handling
  - [x] ActionTxQuery Schema Updates

### PR DESCRIPTION
**Pull request by Akamig**
This resolves #2179.

....Almost There!

TODO:
- [x] Add ApprovePledge Logic Branch
- [x] UI Polishing
- [x] Meow Meow MEOW
- [x] Annihilate CommonJS
- [x] Disturbing Fabric of Space-Time
- [x] Keep Adding Meaningless TODO Item

### CODE REVIEW

File: src/renderer/views/RegisterView/ActivationWaitView.tsx [hunk 1]
```diff
@@ -8,47 +8,34 @@ import H1 from "src/renderer/components/ui/H1";
...
   useEffect(() => {
     (async () => {
-      const activationResult = await activate();
-
-      if (!activationResult.result) {
+      const result = await pledge();
+      console.log(result.result);
+      if (result.result === true) {
+        history.push("/register/activationSuccess");
+      } else if (result.result === false) {
```
Comment (longfin) [inline src/renderer/views/RegisterView/ActivationWaitView.tsx:27]:
```suggestion
      const { result } = await pledge();
      if (!!result) {
        history.push("/register/activationSuccess");
      } else {
```

File: src/utils/usePledge.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          case "STAGING":
+            break;
+        }
+        await sleep(1000);
+      }
+
+      step = "createApprovePledgeTx";
+      await approve({
+        variables: {
+          publicKey: account.loginSession.publicKey.toHex("compressed"),
```
Comment (longfin) [inline src/utils/usePledge.ts:93]: AFIK, Libplanet requires an uncompressed form of public keys to sign.  (see also: https://github.com/planetarium/libplanet/blob/1.2.1/Libplanet/Tx/TxMarshaler.cs#L61)

File: src/api.graphql [hunk 3]
```diff
@@ -445,6 +422,16 @@ query MigrateMonsterCollection($publicKey: String!, $avatarAddress: Address) {
...
+  stateQuery{
+    contracted(agentAddress: $agentAddress){
+			contracted
+    }
+  }
+}
+
+query approvePledge($publicKey: String!){
+	actionTxQuery(publicKey: $publicKey){
+    approvePledge(patronAddress: "0x441Ad1fD834BaB75c6E5859e5Bf736c97Ffa6cc5")
```
Comment (longfin) [inline src/api.graphql:435]: I think that maintaining this address as constant or configurable might be better than here. (it's the place hard to guess.)

**Review by longfin — APPROVED** [review]

**Comment by Akamig** [comment]
Just going with #2200!

**Pull request closed without merging**